internal/rootcoord/meta_table.go

```diff
@@ -205,17 +230,18 @@ func (mt *MetaTable) ChangeCollectionState(ctx context.Context, collectionID Uni
 	mt.ddLock.Lock()
 	defer mt.ddLock.Unlock()
 
-	coll, ok := mt.collID2Meta[collectionID]
-	if !ok {
-		return nil
+	coll, err := mt.collID2Meta.Get(collectionID)
```

Comment:
Will `Get` pull the collection info from catalog? If not, should we explicitly get it from catalog if cache missed?

Reply:
Yes, it will load from catalog if the cache missed